Launcher/Classes/ViewControllers/InspectorViewController.swift

```diff
@@ -250,18 +231,23 @@ class InspectorViewController: NSViewController, NSTableViewDataSource {
     
     func startDevice() {
         try? fileManager.removeItem(atPath: "/tmp/screenshot_0.png")
-        let taskQueue8 = DispatchQueue.global(qos: .background)
         
-        taskQueue8.async {
-            let path = Constants.FilePaths.Bash.startDevice
-            self.runDeviceTask2 = Process()
-            self.runDeviceTask2.launchPath = path
+        if let launchPath = Constants.FilePaths.Bash.startDevice {
+            let outputStream = CommandsCore.CommandTextOutputStream()
+            outputStream.textHandler = { text in
+                if !text.isEmpty {
+                    DispatchQueue.main.async {
+                        self.textViewPrinter.printToTextView(text)
+                    }
+                }
+            }
+            var arguments: [String] = []
             if let phoneUDID = self.applicationStateHandler.phoneUDID {
-                self.runDeviceTask2.arguments = [phoneUDID]
+                arguments = [phoneUDID]
+            }
+            DispatchQueue.global(qos: .background).async {
+                self.commands.executeCommand(at: launchPath, arguments: arguments, outputStream: outputStream)
```

Comment:
Why is this run on a background thread (but eg. https://github.com/JoeSSS/calabash-launcher/pull/31/files#diff-8622fb2ab735c45bc8f54aee4b135938R223 isn't?)

Reply:
There are some methods like running tests or this one that takes a lot of time to continue. If I do it in the main thread, it will freeze the whole APP, but we actually can use the APP while these commands are executing in background